src/feedspora/__main__.py

```diff
@@ -28,26 +29,42 @@ def read_config_file(filename):
 
 
 if __name__ == '__main__':
-    config = read_config_file('feedspora.yml')
+    # Parse input args
+    parser = argparse.ArgumentParser(
+        description='Post from Atom/RSS feeds to various client types.')
+    parser.add_argument('-t', '--testing', nargs='?',
+                        const='feedspora', default=None,
+                        help='execute test runs; no actual posting done')
+    args = parser.parse_args()
+
+    # root name of config and DB files, optionally modified by the --testing
+    # argument value (if present)
+    root_name = 'feedspora'
```

Comment:
The ternary operator would be nice: `root_name = args.testing if args.testing else 'feedspora'` (or maybe 'feedspora' could be the default value for parameter `testing`?) What do you think?